pages/en/lb3/Attaching-models-to-data-sources.md

```diff
@@ -55,9 +55,26 @@ The tool will then add an entry such as the following to `datasources.json`:
 
 This example creates a MongoDB data source called "mongo1". The identifier determines the name by which you refer to the data source and can be any string.
 
+Next, the generator will prompt you to type the datasource settings, such as
+host, port, user, password and database name, as well as to install the database connector.
+
+```shell
+? Enter the datasource name: mongo1
+? Select the connector for mongodb: MongoDB (supported by StrongLoop)
```

Comment:
Change this to `Select the connector for mongo1` instead of `Select the connector for mongodb`